CONTRIBUTING.md

```
@@ -57,30 +85,95 @@ Make sure you have read the [Documentation for Setting up the Project](https://g
+         ```
+
+         You don't need to re-run the `npm run jest-preview` command each time, simply run the `npm run test` command if the Jest Preview server is already running in the background, it'll automatically detect any failing tests and show the preview at `http://localhost:3336` as shown in this screenshot -
+
+         ![Debugging Test Demo](./public/images/jest-preview.webp)
+
+   3. **_Test Code Coverage_:**
+
+      1. _General Information_
+         1. The current code coverage of the repo is: [![codecov](https://codecov.io/gh/PalisadoesFoundation/talawa-admin/branch/develop/graph/badge.svg?token=II0R0RREES)](https://codecov.io/gh/PalisadoesFoundation/talawa-admin)
+         2. You can determine the percentage test coverage of your code by running these two commands in sequence:
+            ```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)